dialogue/TownNPCs/Olive/RandomCardinalAzuraBanana/CardinalAzuraBanana_Olive_Random2.json

```diff
@@ -11,11 +11,17 @@
       "Things like its wildlife seeming very cute at first, but actually being quite dangerous!",
       "I have no idea how she knew, though... Maybe she was studying the outside world and I just never noticed?"
     ],
+    "pl": [
+      "Las to niebezpieczne miejsce... Gdy byłyśmy młode, moja siostra czasami opowiadała o nim.",
```

Comment:
I think you should use "małe" instead of "młode", because it sounds better in this context